### ISSUE TITLE
⌨ 

### ISSUE THREAD

**Issue by RM-1978** [GCTC-NTGC/gc-digital-talent#8081]
## ⌨ Accessibility Issue

Manual accessibility review 

### WCAG 2.1 Criteria

List any relevant criteria that is failing.

## 🦋 Expected Behaviour

What should be happening?

## 🕵️ Details

Add any additional details that could assist with troubleshooting/fixing the issue.

- **Operating System**: Name and version (if possible).
- **Browser**: Name and version (if possible).
- **Assistive Technology**: Name and version (if possible).

## 📋 Steps to Reproduce

1. ...
2. ...

## 📸 Screenshot

Add a screenshot (if possible).

## 🙋‍♀️ Proposed Solution

(optional) Do you have a proposed solution?

## ✅ Acceptance Criteria

A set of assumptions which, when tested, verify that the issue was addressed.

- [ ] Criteria 1
- [ ] Criteria 2

**Comment by esizer** [comment]
Looks like this one was made on accident. Closing.
